packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -44,6 +47,28 @@ interface Props {
 
 const getTemplateValue = (template: {name: string}) => template.name
 
+const CATEGORY_ID_TO_NAME = {
+  recommended: 'Quick Start',
+  retro: 'Retrospective',
+  estimation: 'Estimation',
+  standup: 'Standup',
+  feedback: 'Feedback',
+  strategy: 'Strategy'
+}
+
+type CategoryID = keyof typeof CATEGORY_ID_TO_NAME
+
+// :TODO: (jmtaber129): Fold this into the 'MeetingThemes' to be added in
+// https://github.com/ParabolInc/parabol/pull/7908.
+const CATEGORY_ID_TO_COLOR_CLASS = {
+  recommended: 'bg-grape-700',
+  retro: 'bg-grape-500',
+  estimation: 'bg-tomato-500',
+  standup: 'bg-aqua-400',
+  feedback: 'bg-jade-400',
+  strategy: 'bg-rose-500'
+}
```

Comment:
This is duplicated in part in https://github.com/ParabolInc/parabol/pull/7908, so we should merge the two mappings either as part of whichever PR lands last, or after both PRs land.